semrel.go

```diff
@@ -249,9 +249,10 @@ func CalculateChange(commits []*Commit, latestRelease *Release) Change {
 }
 
 func ApplyChange(latestVersion *semver.Version, prerelease string, change Change) *semver.Version {
-	if latestVersion.Major() == 0 {
-		change.Major = true
-	}
+	// Don't force things to 1.0
```

Comment:
Think better to "just" remove the code if we don't want it.

Reply:
Yeah, I was waffling on that ... this is upstream code, was pondering if we ever tried to sync with upstream, might be helpful for changes to be clearer, but in retrospect, this is a silly place to do that

Reply:
I've tried to sync with upstream a couple of times. Pretty futile in my opinion. It'd be easier to re-fork and add what we want back into that vs merging them into us.